<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color: qlineargradient(spread:pad, x1:0.2441, y1:0.318, x2:0.263682, y2:1, stop:0.079602 rgba(84, 233, 255, 255), stop:0.636816 rgba(33, 194, 255, 255), stop:1 rgba(14, 179, 255, 255));}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>481</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 italic 48pt &quot;Cascadia Mono SemiBold&quot;;
color: rgb(84, 158, 255);
</string>
    </property>
    <property name="text">
     <string>Inventario</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablaInventario">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>1181</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <column>
     <property name="text">
      <string>C칩digo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Producto</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Proveedor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descripci칩n</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Stock</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Precio Unitario</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Producci칩n</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Vencimiento</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="atrasBoton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>700</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;
border-radius:20px;
background-color: rgb(255, 255, 255);

font: 15pt &quot;MS Shell Dlg 2&quot;;
color: rgb(37, 70, 180);

</string>
    </property>
    <property name="text">
     <string>Atr치s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reflescarBoton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>700</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;
border-radius:20px;
background-color: rgb(255, 255, 255);

font: 15pt &quot;MS Shell Dlg 2&quot;;
color: rgb(37, 70, 180);

</string>
    </property>
    <property name="text">
     <string>Reflescar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
